server/models/Message.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const MessageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user',
+      required: true,
+    },
+    recipient: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user',
+      required: true,
+    },
+    type: {
+      type: String,
+      enum: ['msg', 'img'],
+    },
+    read: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    sendDate: {
```

Comment:
We don't need sendDate here as we the timestamps will provide the creation dates for you :)

Reply:
I forgot about that. Thanks.